structure/fenwicktree/fenwicktree_test.go

```diff
@@ -0,0 +1,80 @@
+package fenwicktree
```

Comment:
Consider writing a black box test for this...

```suggestion
package fenwicktree_test
```